Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+FROM golang:1.10 AS builder
 WORKDIR /go/src/github.com/NVIDIA/gpu-operator
 COPY . .
 RUN make build
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+FROM debian:stable-slim
```

Comment:
Followup (commit): Can we have the same architecture as https://github.com/NVIDIA/k8s-device-plugin
So that we have an image based on openshift.